assisted_deployment.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+TEST_INFRA_BRANCH=${TEST_INFRA_BRANCH:-"master"}
+TEST_INFRA_DIR=$WORKING_DIR/test-infra
+
+
+function install_assisted_env() {
+  pushd $WORKING_DIR
+  if cd $TEST_INFRA_DIR;then
+    git fetch --all && git reset --hard origin/$TEST_INFRA_BRANCH;
+  else
+    git clone --branch $TEST_INFRA_BRANCH https://github.com/tsorya/test-infra.git;
```

Comment:
Could we move all the assisted  install stuff to the openshift or openshift-metal3 orgs?

Reply:
When i will know where ot move, surely i will